FirebaseCore/Extension/FIRLogger.h

```diff
@@ -167,7 +167,7 @@ NS_SWIFT_NAME(FirebaseLogger)
 ///   - message: Formatted string to be used as the log's message.
 ///   - args: Arguments list obtained from calling `va_start`, used when message is a format string.
 + (void)logWithLevel:(FIRLoggerLevel)level
-         withService:(FIRLoggerService)service
+         withService:(NSString *)category
             withCode:(NSString *)messageCode
          withMessage:(NSString *)message
             withArgs:(va_list)args;
```

Comment:
Should we do the `category` vs. `service` cleanup now or in a future release? I _think_ it'll only break this line: https://github.com/firebase/firebase-ios-sdk/blob/96890a38535a482d468970cd971b4e118de80112/FirebaseMLModelDownloader/Sources/DeviceLogger.swift#L79

If we do it now, please remove the extraneous `with`s at the same time.

Reply:
Scratch that, it's used in Storage, Auth, Firestore too.